frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -70,6 +80,24 @@ const TeamEventCreditForm: React.FC = () => {
         contentMsg: 'Team events must be logged for at least 0.5 hours!'
       });
     } else {
+
+      images.map(async (image, index) => {
+        const newTeamEventCreditAttendance: TeamEventAttendance = {
+          member: userInfo,
+          hoursAttended: teamEvent.hasHours ? Number(hours) : undefined,
+          image: `eventProofs/${getNetIDFromEmail(userInfo.email)}/${new Date().toISOString()}`,
+          eventUuid: teamEvent.uuid,
+          pending: true,
```

Comment:
Type checker should catch this: we killed off the `pending` field already in https://github.com/cornell-dti/idol/pull/575.